src/commands/authorize.ts

```diff
@@ -1,12 +1,11 @@
-import * as zod from 'zod';
-import {Commands, Scopes} from '../schema/common';
+import {Commands} from '../schema/common';
 import {AuthorizeResponse} from '../schema/responses';
-import {TSendCommand} from '../schema/types';
+import {OAuthScopes, TSendCommand} from '../schema/types';
 import {commandFactory} from '../utils/commandFactory';
 
 export interface AuthorizeInput {
   client_id: string;
-  scope: Array<zod.infer<typeof Scopes>>;
+  scope: OAuthScopes[];
```

Comment:
Oh... it looks like the previous type here was inferring the list of scopes from a generated file [here](https://github.com/discord/embedded-app-sdk/blob/46cf04e89a21194eb15425d64a1ad3b1f25461a7/src/generated/schemas.ts#L55) that did include `applications.commands`. Is it actually better to use a separate list of OAuth scopes here? Or should we lean on the generated definition and move the `zod.infer` into types.ts (it looks like lots of exported types there follow this pattern).

Reply:
personally I prefer this approach, it's much easier to understand and some of the scopes that were inferred from the generated type are actually not grantable to a majority of applications

Reply:
While I agree that using generated types isn't as clear, I think the best solution may be to introduce a cleanup with v2 instead of via this PR, as reformatting things may introduce unintended breaking changes

Reply:
I think the swap done in this line is okay, but we should refrain from deleting the other exports.